src/WooCommerce.php

```diff
@@ -128,6 +128,148 @@ public static function woocommerce_single_product_image_thumbnail_html($html, $t
     return preg_replace('/(<a\s+)/i', '<a data-srcset="' . $srcset . '" data-sizes="' . $srcsizes . '" ', $html);
   }
 
+  /**
+   * Adds additional images from product variation gallery.
+   *
+   * @implements woocommerce_product_get_gallery_image_ids
+   */
+  public static function woocommerce_product_get_gallery_image_ids($value, $obj) {
+    global $product;
+
+    if ($product->get_type() !== 'variable') {
```

Comment:
The following error is reported by Tideways (in Slack #tideways) for several days already:

> Call to a member function get_type() on null
> in /wp-content/plugins/gallerya/src/WooCommerce.php:139

```suggestion
    if (!$product || $product->get_type() !== 'variable') {
```